testing/cffi1/test_re_python.py

```diff
@@ -238,6 +239,7 @@ def test_partial_enum():
     pytest.raises(VerificationMissing, ffi.emit_python_code,
                    str(tmpdir.join('test_partial_enum.py')))
 
+@pytest.mark.thread_unsafe
```

Comment:
why this one?

Reply:
When multiple threads run this test it errors out with `testing/cffi1/test_re_python.py::test_anonymous_union_inside_struct - ffi.error: don't know the size of ctype 'struct with_struct_with_union'` because type is not done initializing in one thread and sizeof doesn't force initialization